src/main/java/org/apache/sysds/runtime/compress/colgroup/ColGroupFactory.java

```diff
@@ -254,26 +258,51 @@ else if(g instanceof ColGroupSDC)
 
 	private AColGroup compress(CompressedSizeInfoColGroup cg) throws Exception {
 		final IColIndex colIndexes = cg.getColumns();
-		final CompressionType ct = cg.getBestCompressionType();
+		final CompressionType ct = cg.getBestCompressionType(); // TODO: handle quantization-fused compression
```

Comment:
the best compression, should already at this point be correct subject to quantization, assuming the estimators behave correctly.
Therefore, remove this TODO.